test/unit/testtest_spec.lua

```diff
@@ -0,0 +1,19 @@
+local helpers = require('test.unit.helpers')(after_each)
+local assert = require('luassert')
+
+local itp = helpers.gen_itp(it)
+
+local sc = helpers.sc
+
+-- All of the below tests must fail. Check how exactly they fail.
+if os.getenv('NVIM_TEST_RUN_TESTTEST') ~= '1' then
+  return
+end
```

Comment:
Let's just put `return` here, with a comment. That avoids another env var. commenting-out the `return` is the same amount of temporary work as setting `NVIM_TEST_RUN_TESTTEST`, with less noise in the documentation (and environment)